eng/native/build-commons.sh

```diff
@@ -236,12 +236,12 @@ __BuildOS=$os
 __OutputRid=''
 
 # Get the number of processors available to the scheduler
-platform="$(uname)"
-if [[ "$platform" == "FreeBSD" ]]; then
+platform="$(uname -s | tr '[A-Z]' '[a-z]')"
```

Comment:
for consistency with other usages:

```suggestion
platform="$(uname -s | tr '[:upper:]' '[:lower:]')"
```